### ISSUE TITLE
"Space" as input key is ignored

### ISSUE THREAD

**Issue by ligazetom** [godotengine/godot#35576]
Godot version:
3.2.beta.custom_build.24281cd9c

OS/device including version:
Windows 10 Pro (Version: 1903, Build 18362.535)

**Issue description:**
Keyboard's "Space" is ignored on input

**Steps to reproduce:**
Create InputMap.
![image](https://user-images.githubusercontent.com/11858829/73132585-586d2a80-401d-11ea-98ef-989fb2c5e6e9.png)

I print to console everytime "Impulse" is pressed. It works for "M" but not for "Space". 
**Minimal reproduction project:**
I am working with GDNative so it would be cumbersome. However if "M" works and "Space" don't, it must be somewhere deep down.

My "Space" works, since I wrote this with spaces.

**Comment by akien-mga** [comment]
Isn't it because it's already used by `ui_accept`?

**Comment by ligazetom** [comment]
I thought as well, so I deleted it from there and it still kept ignoring it.

**Comment by nathanwfranke** [comment]
Cannot reproduce on 3.2 RC 3 (Mono), Linux Mint 19.3

Here is a reproduction project  
[Test4.zip](https://github.com/godotengine/godot/files/4116111/Test4.zip)

**Comment by ShlomiRex** [comment]
Please show the code that you wrote, I can't reproduce it on Ubuntu 20.04 build 3.1 stable

**Comment by ligazetom** [comment]
Hey, here I shared my whole project: https://github.com/godotengine/godot/issues/35548

I cannot provide you with the code because I came across so many bugs I just got fed up with everything and deleted everything godot related. 

But it was really simple. In _unhandled_input() I had something like:

```
if(event->is_action("Impulse")
{
    std::cout << "Impulse" << std::endl;
}
```
And it was printing impulse when I pressed M, but not Space. Sorry guys for not having source anymore, I just gave up on this engine.
